assets/js/instant-checkout.js

```diff
@@ -378,12 +378,10 @@ jQuery( function( $ ) {
             console.log( 'updateOrder' );
             let fields = $('.woocommerce-checkout').serialize();
 
-            if ( typeof compatibility === 'undefined' ) {
-                compatibility = false;
+            if ( compatibility ) {
+                alert( 'Order update is not available.' );
```

Comment:
Is a browser `alert` the best we can do here?

Reply:
@asbjornu Actually there is `compatibility` should be always false, so customer shouldn't have this alert. I removed compatibility layer when order update wasn't available by swedbank pay api